gtsfm/scene_optimizer.py

```diff
@@ -410,9 +406,35 @@ def save_visualizations(
     return viz_graph_list
 
 
+def get_gt_gtsfm_data(
+    cameras_gt: List[Optional[gtsfm_types.CAMERA_TYPE]],
```

Comment:
the GT cameras should be all or nothing for each of our datasets (either for every image, or none of them)